docs/Contributing/API-Versioning.md

```diff
@@ -107,3 +107,4 @@ This will mean that the following are the only valid paths after this point:
 And the code doesn't have to specify `.StartingAtVersion("2021-12")` anymore.
 
 <meta name="pageOrderInSection" value="900">
+<meta name="description" value="This page contains information on how and why the Fleet API is versioned.">
```

Comment:
Thinking about this in the context of a link preview, I'd get rid of any instance of "This page contains" and replace with "Learn about..."/"Read about..." or just omit entirely. Several change suggestions incoming!
```suggestion
<meta name="description" value="Learn about how and why the Fleet API is versioned.">
```